src/REPLMode.jl

```diff
@@ -1302,7 +1323,7 @@ end
 # TODO remove this
 command_specs = super_specs["package"]
 
-const help = md"""
+help = md"""
```

Comment:
Why remove `const`? `const`is about the constness of the binding but the object can still be mutated.

Reply:
Thanks! I didn't know that